src/File/Size.php

```diff
@@ -87,7 +87,7 @@ public function inBytesPostfix(string $concatenator = ''): string
      */
     public function inKiloBytes(int $decimal = 2): float
     {
-        return number_format($this->size / static::KB_MULTIPLY_VALUE, $decimal);
+        return (float) number_format($this->size / static::KB_MULTIPLY_VALUE, $decimal);
```

Comment:
I think the casting `float` type is necessary because it can let developers know `inKiloBytes` will return `float` type.

If we don't use this casting type syntax, developers will be confused about return type because of `number_format` function.

Even the `:float` type will cast result value type automatically.

Reply:
Please remove space between the cast and `number_format` i.e. `(float)number_format`